pkg/exchange/max/exchange.go

```diff
@@ -268,6 +268,10 @@ func (e *Exchange) QueryOpenOrders(ctx context.Context, symbol string) (orders [
 // lastOrderID is not supported on MAX
 func (e *Exchange) QueryClosedOrders(ctx context.Context, symbol string, since, until time.Time, lastOrderID uint64) ([]types.Order, error) {
 	log.Warn("!!!MAX EXCHANGE API NOTICE!!! the since/until conditions will not be effected on closed orders query, max exchange does not support time-range-based query")
+	if !since.IsZero() || !until.IsZero() {
```

Comment:
should be until.IsZero()? since the api only support one timestamp parameter?

Reply:
We can give one of it. If we give until, we will use 2018-01-01 as timestamp to query and filter it by until.